iml-agent/src/env.rs

```diff
@@ -59,43 +50,24 @@ pub fn mailbox_sock(mailbox: &str) -> String {
 }
 
 lazy_static! {
-    // Gets the pfx file.
-    // If pfx is not found it will be created.
-    pub static ref PFX: Vec<u8> = {
+    pub static ref PEM: Vec<u8> = {
+        let mut result = Vec::new();
+
         let private_pem_path = get_private_pem_path();
 
-        if !path_exists(&private_pem_path) {
-            panic!("{} does not exist", private_pem_path)
-        };
+        let mut private_pem = File::open(private_pem_path)
+            .unwrap_or_else(|_| panic!("{} does not exist", get_private_pem_path()));
```

Comment:
Technically, I think, there could be other errors than "not exist".
It would probably be better to sue the argument of `unwrap_or_else` and print actual error.

Also maybe use `private_pem_path` instead of calling `get_private_pem_path()`?